src/marketplace.ts

```diff
@@ -90,6 +96,110 @@ export class MarketplaceClient extends Client {
   listings(auctionHouse: AuctionHouse): ListingsClient {
     return new ListingsClient(this.connection, this.wallet, auctionHouse)
   }
+
+  sales(auctionHouse: AuctionHouse): SalesClient {
+    return new SalesClient(this.connection, this.wallet, auctionHouse)
+  }
+
+  admin(): AdminClient {
+    return new AdminClient(this.connection, this.wallet)
+  }
+
+  async createAuctionHouses(
```

Comment:
Lets wait on adding this. Do the auction house creation in markeplaces dapp. Great job on putting together the instruction set.